assets/src/eejs/routes.js

```diff
@@ -99,6 +99,7 @@ export const ADMIN_ROUTE_ACTIONS = {
 	VENUES: {
 		OVERVIEW: ADMIN_ROUTE_ACTION_DEFAULT,
 		CATEGORIES: 'category_list',
+		EDIT: 'edit',
```

Comment:
I know it's a pain, but I think this should be done separately in a branch off master.

Reply:
see: #1364